src/ReTestItems.jl

```diff
@@ -563,11 +570,16 @@ function manage_worker(
             @debugv 2 "Error" exception=e
             # Handle the exception
             if e isa TimeoutException
+                @warn "$worker timed out running test item $(repr(testitem.name)) after $timeout seconds."
                 if timeout_profile_wait > 0
-                    @warn "$worker timed out running test item $(repr(testitem.name)) after $timeout seconds. \
-                        A CPU profile will be triggered on the worker and then it will be terminated."
+                    @warn "Gathering a CPU profile from the worker."
```

Comment:
Now that we're logging multiple messages, they might be interspersed with logs coming from other workers, so i think we should always interpolate the `$worker` value so we can identify which worker is being referred to

Also i think the messages after the initial "timeout" log, can be `info` since they're "normal" behaviour for the situation, and it can be nice to keep the number of `warn` messages equal to the number of things that have gone wrong (e.g. timeouts)

```suggestion
                    @info "Gathering a CPU profile from $worker."
```